docs/developer_manual/services/developing_an_assemblyline_service.md

```diff
@@ -32,6 +32,11 @@ Put the following code in your service's file:
 
             self.log.info(f"start() from {self.service_attributes.name} service called")
 
+        def load_rules(self) -> None:
```

Comment:
The updater should not be part of the main service development part since it is not required. It should be in its own section with more detailed instructions. Maybe a section like: **Adding an updater to your service**. 

This detailed section would define:

- What a service updater is
- What type of service need or doesn't need one
- ...